### DISCUSSION TITLE
Where to act for some messages of  x265 [WARN]

### DISCUSSION THREAD

**Discussion by Victory61** [staxrip/staxrip#826] [Q&A] [answered]
Good morning.
I'm performing UHD encoding in x265, I get these following warning messages:

x265 [WARN]: Turning on repeat-headers for HDR compatibility
x265 [WARN]: Specifying a decoder level with constant rate factor rate-control requires
x265 [WARN]: enabling VBV with vbv-bufsize=160000kb vbv-maxrate=160000kbps. VBV outputs are non-deterministic!

In which section should I act?
Thank you very much

**Comment by Hellboy00000** [comment]
I am seeing this warning too "x265 [warning]: Turning on repeat-headers for HDR compatibility".

Someone know what this mean?

**Reply by Dendraspis** [reply, marked as answer]
It means you should turn `repeat-headers` on to get best HDR compatibility in case you really deal with an HDR video and you want to keep it HDR.

**Reply by Peronia** [reply]
Hi @Dendraspis, sorry to bother you. I have exactly the same warning: `x265 [warning]: Turning on repeat-headers for HDR compatibility` but I don't know exactly where I must turn it on. I tried this, but it seems not to work (the message keeps printing):
![grafik](https://github.com/user-attachments/assets/67f517ce-a66f-47f8-a7ff-79a6a74ada72)

**Reply by Dendraspis** [reply]
This isn't even StaxRip, so why do you ask me?
`--repeat-headers` could work...

**Reply by Peronia** [reply]
Yes, you are right. I don't saw that this isn't handbrake. For sure I have to ask there and not here. 😅
